components/blitz/src/omero/gateway/util/Requests.java

```diff
@@ -2223,6 +2246,22 @@ public Chown2Builder toUser(Experimenter user) {
         public Chown2Builder toGroup(Experimenter user) {
             return toUser(user);
         }
+
+        /**
+         * @param ids the IDs of users whose data is to be targeted for this operation, does not overwrite previous calls
+         * @return this builder, for method chaining
+         */
+        public Chown2Builder targetUsers(Long... ids) {
+            return targetUsers(Arrays.asList(ids));
+        }
+
+        /**
+         * @param user the user whose data is to be targeted for this operation, does not overwrite previous calls
+         * @return this builder, for method chaining
+         */
+        public Chown2Builder targetUsers(Experimenter user) {
```

Comment:
I'd say so, the Javadoc is quite clear that we expect a user! We can add a bunch of explicit not-null notes but would prefer to do so in a separate PR because there are plenty of other such methods.